Apps/Sandcastle/gallery/Terrain.html

```diff
@@ -98,7 +99,7 @@
 Sandcastle.addToolbarMenu([{
     text : 'Mount Everest',
     onselect : function() {
-        if (scene.mode === Cesium.SceneMode.SCENE3D || scene.mode === Cesium.SceneMode.COLUMBUS_VIEW) {
+        if (viewer.scene.mode === Cesium.SceneMode.SCENE3D || viewer.scene.mode === Cesium.SceneMode.COLUMBUS_VIEW) {
```

Comment:
We actually don't need these `if` checks anymore. `lookAt` works in all camera modes.  We should remove the if checks and the else blocks.